### ISSUE TITLE
Patch 1.1.4 update seems to be breaking mocha dependency

### ISSUE THREAD

**Issue by bal360** [glenjamin/mocha-multi#82]
Apologies ahead of time if this is a `mocha` issue, rather than an issue with `mocha-multi`, but it appears the recent update to `1.1.4` is now breaking with a range of `mocha` versions. 

**_Dependencies_**: 
```
"dependencies": {
    "@babel/core": "^7.4.5",
    "@babel/preset-env": "^7.4.5",
    "chai": "^4.2.0",
    "jsdom": "^9.0",
    "mocha": "5.2.0",
    "mocha-jsdom": "~1.1.0",
    "mocha-multi": "^1.0.1",
    "sinon": "^7.3.1"
  }
  ```
  
 **_Error:_**
 ```
 bal@Blakes-MacBook-Pro-2 phase-0-intro-to-js-2-looping-code-along % npm install
 npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: js-looping-and-iteration-looping-code-along@0.1.0
npm ERR! Found: mocha@5.2.0
npm ERR! node_modules/mocha
npm ERR!   mocha@"5.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer mocha@"2.2.0-7.0.0 || ^9.0.0" from mocha-multi@1.1.4
npm ERR! node_modules/mocha-multi
npm ERR!   mocha-multi@"^1.0.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/bal/.npm/eresolve-report.txt for a full report.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bal/.npm/_logs/2021-09-30T15_00_58_403Z-debug.log
```

**Comment by glenjamin** [comment]
oof, it seems like #79 might be using some semver syntax that doesn't match? otherwise `5.2.0` should match `2.2.0-7.0.0`?

I'll deprecate 1.1.4 for now until we know what's up

**Comment by glenjamin** [comment]
Ah ok, I've now also manually set the `latest` tag to be `1.1.3` - that should be better?

**Comment by MarcCelani-at** [comment]
doh, sorry about this folks

**Comment by ihollander** [comment]
I think I have a fix in #83 - it looks like a small syntax issue with setting up a range for `2.2.0 - 7.0.0`.

**Comment by glenjamin** [comment]
Ok, this should be fixed in `1.1.5` now.

Thanks for the quick responses everyone!

**Comment by bal360** [comment]
Working for me too! Thanks to everyone for being so prompt!
